.github/workflows/e2e-tests.yaml

```
@@ -20,12 +20,12 @@ jobs:
     if: ${{ github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.draft == false) }}
     steps:
         - name: Checkout code
-          uses: actions/checkout@v3
+          uses: actions/checkout@v5
           with:
             submodules: recursive
 
         - name: Set up Go
-          uses: actions/setup-go@v3
+          uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)